src/compiler/commandLineParser.ts

```diff
@@ -3268,6 +3268,9 @@ function parseOwnConfigOfJson(
     json.compileOnSave = convertCompileOnSaveOptionFromJson(json, basePath, errors);
     let extendedConfigPath: string | string[] | undefined;
 
+    if (json.extends === "") {

```

Comment:
You should reject a value of `json.extends` being `""` as well.

(also, while you're here, can you fix the indentation on the `}` in the below `for` loop?

Reply:
I guess you are doing this check in the tests - but I don't see where the error would be issued.

Reply:
This is where the error would show up. Should there be a compiler test for this too?
![image](https://user-images.githubusercontent.com/108360753/208784991-10a3d822-43e2-452b-9b63-1ab67658b838.png)

Reply:
Sorry, my example wasn't clear.

So I was concerned with the case of an array value containing `""`, such as:

```json5
"extends": [""]
```

or:

```json5
"extends": ["./foo", "", ""]
```

But it already seems like you issue that error from `getExtendsConfigPath`. Wouldn't the case of

```json5
"extends": ""
```

also be issued from `getExtendsConfigPath`?

Reply:
@DanielRosenwasser In the tests for ` "extends": "" `, when it is a `jsonSourceFile` then it enters the function `parseOwnConfigOfJsonSourceFile` where it calls `getExtendsConfigPath` and so it throws an error but when it is a `json` file, then it enters the function `parseOwnConfigOfJson`. Here the value of `extends` is "" so it never reaches a point where it makes a call to `getExtendsConfigPath`. 
So it needs to check if the extends conditions is an empty string and then call `getExtendsConfigPath`